src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin set-colums($min-width, $colums) {
+  @media (min-width: $min-width) {
+    grid-template-columns: repeat($colums, 200px);
```

Comment:
There is a typo in the variable name '$colums'. It should be '$columns' to match the corrected parameter name.